src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/customer_groups.yml

```diff
@@ -0,0 +1,37 @@
+admin_customer_groups_index:
+  path: /
+  methods: [ GET ]
+  defaults:
+    _controller: 'PrestaShopBundle\Controller\Admin\Configure\ShopParameters\CustomerGroupsController::indexAction'
+    _legacy_controller: AdminGroups
+    _legacy_link: AdminGroups
```

Comment:
Only this one blocking, you can't leave it uncommented or it will replace the current legacy page but it's not fully functional yet nor tested

Reply:
If you want to keep it you need to bind it to a related feature flag But I though it could be done later for the feature flag (maybe won't even be useful if it's finished before the end of v9)
But for now then the comment of this is mandatory